mkdocs.yml

```diff
@@ -74,8 +99,45 @@ nav:
     - Base Application: getting-started/base.md
     - Development setup: getting-started/development.md
     - Preparing source folder: getting-started/prepare-source-folder.md
+  - Reference Documentation:
+    - TL;DR: reference-docs/tldr.md
+    - API:
+      - Rest API: reference-docs/api.md
+      - GraphQL: reference-docs/graphql.md
+    - App.py : reference-docs/app/app.md
+    # - Auth(n|z): reference-docs/auth.md
```

Comment:
commented out docs?

I think its better to be able to read them somewhere untill updated. (atleast for this Auth)

Reply:
Agreed, however, these are all placeholders as we don't have any docs written for these yet :(